mysql-test/lib/t/testMyConfig.t

```diff
@@ -40,6 +40,9 @@ my $test_cnf= "$dir/test.cnf";
 open(OUT, ">", $test_cnf) or die;
 
 print OUT <<EOF
+[ENV]
+env_option1=value1
+env_option2= value2
```

Comment:
minor comment, only if you have to change something else too: formatting.

Reply:
This is intentional. Because just below the [ENV] section, we have [mysqld] section which also has a whitespace before the '=' operator to check that MTR actualy trims the whitespaces while processing the cnf file.